package/kernel/mtk-eip93/Makefile

```diff
@@ -0,0 +1,65 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mtk-eip93
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPLv2
+PKG_LICENSE_FILES:=
+
+PKG_SOURCE_URL:=https://github.com/vschagen/mtk-eip93.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=ca08387bf8352652129019bb19e2423ab313d5cb
+PKG_MIRROR_HASH:=4505ee2a175a6cf77a1a3b4fa7bf188da8f8b47894e5afa62c8cb008fbfa15ed
+
+PKG_USE_NINJA:=0
```

Comment:
this is a copy/paste mistake. PKG_USE_NINJA is for cmake.mk only, which is not used here. I'm guessing whoever did this used mt76 as a base...